### ISSUE TITLE
Filter invalid transactions

### ISSUE THREAD

**Issue by nomaxg** [EspressoSystems/op-espresso-integration#3]
Add logic to somehow filter out/check for malformed transactions. Actually we are only worried about malformed, should check to see how this is handled in Arbitrum.

**Comment by jbearer** [comment]
I will close this ticket once https://github.com/EspressoSystems/op-geth/pull/2 is merged, since invalid transactions will be handled. We have a separate ticket for transactions that don't even deserialize, https://github.com/EspressoSystems/op-geth/issues/4, which we probably won't get to this sprint.
